### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [Hainanu-Application/Hainanu-Application.github.io#2] [Announcements]
# index

A magical documentation generator.

https://hainanu-application.github.io/#/

**Comment by yyyanbj** [comment]
Hi, Hainanu Application.

**Reply by polyBK** [reply]
本来也想想谢谢土木相关，太忙了哈哈哈

**Reply by yyyanbj** [reply]
什么时间都欢迎~

**Comment by Vir-Limerence** [comment]
good

**Comment by zouCode** [comment]
nice

**Comment by Plucky923** [comment]
Hello

**Reply by chenzongyao200127** [reply]
Is there anybody in there?

**Reply by Plucky923** [reply]
Hey you

**Comment by peng-yq** [comment]
欢迎投稿，欢迎转发！！！

**Comment by rookie2000** [comment]
火前留名👻

**Comment by JingMingT** [comment]
信息共享很重要呀🛫🛫🛫

**Comment by n0vic30926** [comment]
### 《海南带学自救手册》

**Reply by Plucky923** [reply]
gunna

**Comment by Hammour-steak** [comment]
:heart_eyes:

**Reply by Plucky923** [reply]
狠狠焯你

**Comment by CodeSlogan** [comment]
### 《少走弯路手册》

**Comment by ZhiZI-i** [comment]
想蹲材料的呜呜呜

**Reply by peng-yq** [reply]
呜呜已经很久没人投稿了，我们也宣传了下，愿意投稿的貌似不太多呜呜

**Reply by ZhiZI-i** [reply]
唉慢慢会有人的趴，乃是第四次来看了，感觉还是收获很多

**Reply by peng-yq** [reply]
有收获就好！

**Reply by xiaohai99** [reply]
我感觉投稿得一对一私聊，而且可能需要催几次，才能出来一篇...大范围宣传的转化率还是比较低的（哭泣

**Reply by peng-yq** [reply]
谢谢你的建议，18级的帖子基本都是我们一对一邀请的，有的同学答应了，有的同学答应了但是没有写，也有的同学拒绝了。19级由于没人负责加上我们人脉有限，只能处于大规模宣传的方式。如果同学你愿意负责后续的投稿，欢迎联系我。

**Reply by Payne-zpy** [reply]
材料的有啦

**Comment by yvette707** [comment]
特别特别好，但是感觉github好多人还不知道😿

**Reply by chenzongyao200127** [reply]
嘻嘻有空可以给同学们分享一下~发挥开源精神！

**Comment by shenxiangzhuang** [comment]
挺好的站点。不过通过GitHub能触达的人还是比较少的，可以考虑让在校的学弟学妹们运营一个公众号，然后多做一些线下的宣传。

**Reply by peng-yq** [reply]
是的，现在手册的延续性做的很差，最初的同学们都毕业好久了，没找到接替的同学。
